docs/source/models/gbd2017/causes/neural_tube_defects.rst

```diff
@@ -0,0 +1,75 @@
+.. _2017_cause_neural_tube_defects:
+
+==========================
+Neural Tube Defects (NTDs)
+==========================
+
+Disease Description
+-------------------
+
+Spina bifida corresponds to the ICD-10_ codes Q05.0, Q05.4,Q05.6, Q05.7, Q05.8, and Q05.9. Encephalocele corresponds to the ICD-10 codes Q01.2, Q01.8, and Q01.9. Anencephaly corresponds to the ICD-10 codes Q00.0 and Q00.2. [GBD-2017-YLD-Capstone-Appendix-1]_
```

Comment:
Will we model these separately or all together?

Reply:
Together. They are modeled together as a single cause (neural tube defects) in the GBD, with many, many sequelae describing the different outcomes.